sdk/tvOS/OoyalaTVSkinSDK/Widgets/OOOoyalaTVTopBar.h

```diff
@@ -0,0 +1,21 @@
+//
+//  OOOoyalaTVTopBar.h
+//  OoyalaTVSkinSDK
+//
+//  Created by Ileana Padilla on 7/18/17.
+//  Copyright © 2017 ooyala. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface OOOoyalaTVTopBar : UIView
+
+- (id)initWithBackground:(UIView *)background;
+
+- (id)initMiniView:(UIView *)background;
```

Comment:
this displays a visual indicator to let user know that asset has CCs, happens when video is paused, then user can change focus to this mini view and get the CC menu.